blueprint/platform.md

```diff
@@ -565,9 +565,9 @@ Microsoft Defender for Endpoint integration | Enabled | To provide additional vi
 Enforce app access | Enabled | To block access to apps marked as Unsanctioned.
 List of sanctioned and unsanctioned cloud apps | To be developed during build with the Agency's Cyber Intelligence team. | Provides visibility within the Agency as to what cloud applications are in use and by which department within the Agency.
 
-### MCAS - Log collector
+#### Automatic log upload & Log collectors
```

Comment:
Off our discussion around log collectors, could you please check the implementation applicability for this section and MCAS - Cloud Discovery are accurate?

Reply:
That is, are they still for all implementation types?

Reply:
Have updated the design and pushed another commit.